test/functionalTest/cases/1585_log_summary/log_summary.test

```diff
@@ -176,7 +165,7 @@ Date: REGEX(.*)
  Transactions: 4 (new: 4)
  DB status: ok (created: 0, removed: 0)
  Notification failure active alarms: 0 (created: 0, removed: 0)
- Bad input active alarms: 1 (created: 6, removed: 0)
+ Bad input active alarms: 1 (created: 1, removed: 0)
```

Comment:
Not sure I understand this comment.
Your comment is correct, I just don't see anything to be changed ...